lib/storage/src/types.rs

```diff
@@ -53,7 +53,7 @@ pub struct StorageConfig {
     pub storage_path: String,
     #[serde(default = "default_snapshots_path")]
     #[validate(length(min = 1))]
-    pub snapshots_path: String,
+    pub snapshots_path: Option<String>,
```

Comment:
This change is needed because in the future there could be either snapshot_path for local storage either some details for the storage back end. It would have been cleaner with an enum in here but I believe it would break backwards compatibility.

Reply:
:check:

Reply:
@generall an enum would mean to also have a custom deserialization implementation and everything, overall I felt that it was cleaner how I implemented it here, but for reference this is how it would look like https://github.com/AlexandruCihodaru/qdrant/commit/887d42631996c554e419f8b54eb2b95c397914b6